app/styles/ui/_side-by-side-diff.scss

```diff
@@ -168,6 +188,13 @@
       }
     }
 
+    input:focus-visible + label.hunk-handle {
+      .focus-handle {
+        outline: 2px solid #0659d6;
```

Comment:
Nit: I see we have two instances of `#0659d6` in this file. Should we move this to a var in `_variables.scss`? Is it something we could want to theme depending on light/dark appearance? 🤔

Reply:
Good call out. I will address this in the visual improvements PR since it already deals with the css file.